source/_integrations/alarm_control_panel.manual_mqtt.markdown

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
This file rename no longer matches the domain it belongs to.

Reply:
There is already a precedent of `alarm_control_panel.mqtt` and `alarm_control_panel.template` which to be honest make a lot of sense to me as these are different internal implementations of the same thing.

Reply:
> There is already a precedent

That is legacy.

Reply:
Does it mean that all template docs should live in the same file because they all technically use the same integration? That would be very hard to read and understand, especially since there are so many unrelated template integrations.

But okay, I can move files back and just change the name...

Reply:
> Does it mean that all template docs should live in the same file because they all technically use the same integration? 

Yes.

However, that integration hasn't done that in docs at this point, as the template integration is currently moving from platform to integration level configuration itself. We haven't squashed everything in the docs yet as the code is still in progress.